packages/theming/src/utils/StyledBaseIcon.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import React, { Children } from 'react';
+
+const COMPONENT_ID = 'icon.base';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars,garden-local/require-default-theme
+export const StyledBaseIcon = styled(({ children, theme, ...props }) =>
+  React.cloneElement(Children.only(children), props)
+).attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})`
+  /* stylelint-disable no-empty-block */
+`;
```

Comment:
```suggestion
export const StyledBaseIcon = styled(({ children, theme, ...props }) =>
  React.cloneElement(Children.only(children), props)
)`
  /* stylelint-disable no-empty-block */
`;
```

The `attrs` block should not be needed. Every Garden extension will apply their own. And we shouldn't assume these attributes for any, potential external consumers. Also, the component ID is only used for `retrieveComponentStyles` CSS overrides.